flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "skip-icons-check": true,
+  "only-arches": ["x86_64"]
```

Comment:
Technically, StreamFX supports arm64 if the dependencies are all built for arm64 as well.

Reply:
but is there a flatpak of OBS on aarch64? I don't think.

Reply:
There is not, judging from the [beta.flathub.org page](https://beta.flathub.org/apps/details/com.obsproject.Studio). Maybe upstream could look into building for aarch64 as well?

Reply:
aarch64 builds are planned, but there are challenges that prevented it from happening, and it shouldn't block plugins from being accepted into Flathub. Whenever aarch64 builds are published to Flathub by the OBS community, plugin authors will be notified